src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+
+  const connect = (x) => (y) => {
+    count++;
+
+    if ((count > 3) && (count % 2 === 0)) {
+      return 'Bzzz... Error!';
+    } else {
+      return x + y;
+    }
```

Comment:
use early return pattern 
```suggestion
    }

     return x + y;
```